php/antidot/config.yaml

```diff
@@ -8,9 +8,6 @@ build_deps:
   - libicu-dev
   - libssl-dev
 
-php_ext:
```

Comment:
Removed because it fails on the build process.

![image](https://user-images.githubusercontent.com/1093654/132944633-d7896687-94a5-42df-a859-060352d04838.png)

Reply:
> antidot does not rely on libevent ? I thought it was a requirement for reactphp

React-PHP checks for extensions enabled to select the event loop implementation. When any of the described extensions aren't available, it relies on the `StreamSelectLoop` class, based on the  `stream_select` function. For more info about the different event loop implementations, you can take a look at React-PHP Event Loop Docs https://github.com/reactphp/event-loop#loop-implementations.

Reply:
The warning disappears using the Alpine image, and we can enable the event extension again, thanks;-D